.github/workflows/spread.yml

```diff
@@ -12,6 +12,10 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number && format('pr{0}', github.event.number) || github.run_id }}
```

Comment:
I'm not sure I know how to interpret this incantation, but what harm can come of it?

Reply:
If `github.event.number` is not empty (only in pull requests), use `pr###`, otherwise use `github.run_id` - which will be unique for each run. So outside of pull requests, no cancellations will happen.